app/styles/onboarding-card.module.css

```diff
@@ -36,20 +49,21 @@
     font-style: normal;
     font-weight: 700;
     line-height: 24px; 
-    padding-top: 5rem;
+    padding-top: 3rem;
 }
 
-@media (max-width: 800px) {
+@media (max-width: 1024px) {
     .onboarding-card {
-        min-width: 60vh;
+        width: 80%;
```

Comment:
Why you written width only
not max-width and 
Nit: what is difference between `max-width` and `width`

Reply:
Width sets the exact width of an element, while max-width restricts it from exceeding a certain value